src/Pages/Adopt.jsx

```diff
@@ -67,13 +69,40 @@ export const Adopt = () => {
       vet_address: "",
       vet_phone: "",
       residence_type: "Single Family House",
+      renter_verification: "Not yet",
+      weekend_different: "No",
+      weekend_routine: "",
+      bird_hours_alone: "",
+      smokers_in_house: "No",
+      smokers_explanation: "",
+      other_pets_in_home: "No",
+      what_other_pets: "",
+      what_supp_info: "",
+      lifestyle_changes: "",
+      vacation_care: "",
+      death_plans: "",
+      looking_for_in_bird: "",
+      additional_comments: "",
+      home_address_agreement: false,
+      home_visit_agreement: false,
+      no_guarantee_agreement: false,
     },
   });
   const watchOtherBirds = watch("have_other_birds", "No");
   const watchPreviousBirds = watch("previous_birds", "No");
   const watchAvianVet = watch("have_avian_vet", "No");
+  const watchWeekendDifferent = watch("weekend_different", "No");
+  const watchSmokersInHouse = watch("smokers_in_house", "No");
+  const watchOtherPets = watch("other_pets_in_home", "No");
 
   const onSubmit = (data) => {
+    for (const key of [
```

Comment:
If you're trying to exclude these from submissionData, you could destructure them in the const on line 106 and just not use them.

Reply:
Sounds good! I'll do it this way instead. I was being a bit perfectionistic about avoiding linter squigglies :p

Reply:
Hahaha! Totally understand 😉